### PR TITLE
Tool 954 dev portal data

### PR DESCRIPTION
**Pull request by godrei**
1. devportalservice.SessionData error handled regarding to the error type

2. verbose_log input introduced

### CODE REVIEW

**Comment by godrei** [comment]
Do not merge before this PR: https://github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/pull/66 would be merged and go deps would be updated

File: main.go [hunk 1]
```diff
@@ -102,16 +103,20 @@ func main() {
...
 	// Fastlane session
-	fmt.Println()
-	log.Infof("Ensure cookies for Apple Developer Portal")
-
-	fs, errors := devportalservice.SessionData()
-	if errors != nil {
-		log.Warnf("Failed to activate the Bitrise Apple Developer Portal connection: %s\nRead more: https://devcenter.bitrise.io/getting-started/connecting-apple-dev-account/ \nerrors:")
-		for _, err := range errors {
-			log.Errorf("%s\n", err)
+	fs, err := devportalservice.SessionData()
```
Comment (lpusok) [inline main.go:125]: NOTE: Consider moving the devportal session check to a function.

File: main.go [hunk 2]
```diff
@@ -102,16 +103,20 @@ func main() {
...
+	if err != nil {
+		if networkErr, ok := err.(devportalservice.NetworkError); ok && networkErr.Status == http.StatusNotFound {
+			log.Debugf("")
```
Comment (lpusok) [inline main.go]: Empty line could be printed once, before the `if err != nil`

Comment (godrei) [reply main.go]: but in case of NotFound Network error, it is just a debug log, otherwise a normal log.

File: step.yml [hunk 3]
```diff
@@ -52,3 +52,11 @@ inputs:
       value_options:
       - "true"
       - "false"
+  - verbose_log: "no"
```
Comment (BirmacherAkos) [inline step.yml:49]: please remove the go dependency:
https://github.com/bitrise-steplib/steps-fastlane/blob/fa74b539eafc5b4ab2e984920030933cc875c9bc/step.yml#L16-L21

**Review by BirmacherAkos — CHANGES_REQUESTED** [review]
Please check my notes

**Review by BirmacherAkos — APPROVED** [review]

**Pull request merged**